addons/hr_skills/models/hr_employee.py

```diff
@@ -8,7 +8,7 @@ class Employee(models.Model):
     _inherit = 'hr.employee'
 
     resume_line_ids = fields.One2many('hr.resume.line', 'employee_id', string="Resume lines")
-    employee_skill_ids = fields.One2many('hr.employee.skill', 'employee_id', string="Skills")
+    employee_skill_ids = fields.One2many('hr.employee.skill', 'employee_id', string="Skills", domain=[('skill_type_id.active', '=', True)])
```

Comment:
Usually, we want to keep the length of a code line below 120 characters to keep it readable easily.
If you're using VSCode or VSCodium, you can go in your settings and edit rulers, it will allow you to add lines at certain columns to see easily if the lines you write are too long or not (it's not always possible to respect that limit)
```json
    "editor.rulers": [
        120,
    ],
``` 
```suggestion
    employee_skill_ids = fields.One2many('hr.employee.skill', 'employee_id', string="Skills",
        domain=[('skill_type_id.active', '=', True)])
```